ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -598,30 +600,70 @@ def test014_update_order_material_type(self, save):
                                                                                           order_material_type))
             self.assertEqual(current_material_type, order_material_type)
 
-    # wiill continue with us
-    def test015_filter_by_any_fields(self):
+    @parameterized.expand(['Order No.', 'Contact Name', 'Material Type', 'Test Plans', 
+                           'Test Date', 'Article Name', 'Created On', 'Changed By',
+                           'Shipment Date', 'Status', 'Analysis No.', 'Departments'])
+    def test015_filter_by_any_fields(self, key):
         """
         New: Orders: Filter Approach: I can filter by any field in the table view
         LIMS-3495
         """
-        order_row = self.order_page.get_random_order_row()
-        order_data = self.base_selenium.get_row_cells_dict_related_to_header(
-            row=order_row)
-        filter_fields_dict = self.order_page.order_filters_element()
+        self.info('Create new order')
```

Comment:
I don't know why we need to create new order in this test case, instead we can select any random order ?

Reply:
@ghada-37 I have to make sure that, the selected order has all the fields filled weather required or not required, choosing a random order might not have all the fields filled.

Reply:
ok , I got it